sidebars.js

```diff
@@ -123,11 +123,11 @@ module.exports = {
             'platform-engineers/cue/advanced',
           ]
         },
-        {
-          'Environment System': [
-            'platform-engineers/initializer/basic-initializer',
-          ]
-        },
+        // {
```

Comment:
If it is not used any more, just delete it

Reply:
Sure, before v1.1 release, I'll remove all these redundant ones.